plugins/scaffolder/src/components/Router.tsx

```diff
@@ -15,17 +15,38 @@
  */
 
 import React from 'react';
-import { Routes, Route } from 'react-router';
+import { Routes, Route, useOutlet } from 'react-router';
 import { ScaffolderPage } from './ScaffolderPage';
 import { TemplatePage } from './TemplatePage';
 import { TaskPage } from './TaskPage';
 import { ActionsPage } from './ActionsPage';
+import { getComponentData } from '@backstage/core';
+import { FieldExtensionOptions } from '../extensions';
 
-export const Router = () => (
-  <Routes>
-    <Route path="/" element={<ScaffolderPage />} />
-    <Route path="/templates/:templateName" element={<TemplatePage />} />
-    <Route path="/tasks/:taskId" element={<TaskPage />} />
-    <Route path="/actions" element={<ActionsPage />} />
-  </Routes>
-);
+export const Router = () => {
+  const children = useOutlet();
```

Comment:
This seems to return something unexpected @rugvip. I thought it would return 2 things for 

```tsx
<Route path="/create" element={<ScaffolderPage />}>
      <RepoUrlPickerFieldExtension />
      <OwnerPickerFieldExtension />
</Route>
 ```

But it seems to return 3 items and one of those elements has `props` of these two elements, so I don't think we can rely on it. Maybe we need to use `traverseElementTree` to be safe? Thoughts?